src/pngwriter.cc

```diff
@@ -2023,6 +2023,11 @@ void pngwriter::write_png(void)
 ///////////////////////////////////////////
 void pngwriter::plot_text( char * face_path, int fontsize, int x_start, int y_start, double angle, char * text, double red, double green, double blue)
```

Comment:
Well I didn't want to change the interface, else linking with this library wouldn't be downwards compatible I think.